components/com_users/tmpl/reset/complete.php

```diff
@@ -32,8 +32,11 @@
     <form action="<?php echo Route::_('index.php?task=reset.complete'); ?>" method="post" class="com-users-reset-complete__form form-validate form-horizontal well">
         <?php foreach ($this->form->getFieldsets() as $fieldset) : ?>
             <fieldset>
-                <?php if (isset($fieldset->label)) : ?>
-                    <legend><?php echo Text::_($fieldset->label); ?></legend>
+                <?php if (isset($fieldset->legend)) : ?>
+                    <legend><?php echo Text::_($fieldset->legend); ?></legend>
+                <?php endif; ?>
```

Comment:
```suggestion
                <?php if (isset($fieldset->legend)) : ?>
                    <legend><?php echo Text::_($fieldset->legend); ?></legend>
                <?php elseif (isset($fieldset->label)) : ?>
                    <legend><?php echo Text::_($fieldset->label); ?></legend>
                <?php endif; ?>
```
Question: Should we do it here and in the other tmpl/reset files like this suggestion so it is fully b/c with old XML files e.g. from overrides?

@HLeithner What do you think?

Of course this will not solve the problem mentioned by @chmst in [her comment](https://github.com/joomla/joomla-cms/pull/46454#issuecomment-3583290230) that a fieldset should always have a legend.

Reply:
What if there is neither a legend nor a label? 
In nearly all layouts of the backend we are using a language key for legends. So I think we can take this as a standard.
example 
```
                    <fieldset id="fieldset-publishingdata" class="options-form">
                        <legend><?php echo Text::_('JGLOBAL_FIELDSET_PUBLISHING'); ?></legend>
                        <div>
                        <?php echo LayoutHelper::render('joomla.edit.publishingdata', $this); ?>
                        </div>
                    </fieldset>
```

Reply:
better move it to the xml, so it's configurable

Reply:
your suggestion would force to have a legend, else you can't use the label as "label" for something else on another position it would be doubled.